opm/grid/cpgrid/processEclipseFormat.cpp

```diff
@@ -127,6 +127,16 @@ namespace cpgrid
     {
         std::vector<std::size_t> removed_cells;
         if (ccobj_.rank() != 0 ) {
+            if(ecl_state) {
```

Comment:
Nitpicks: space after `if`, also after `try` on line 166, and on line 194 around `catch` and before the `{`.